types/isomorphic-fetch.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'isomorphic-fetch' {
+  export = window.fetch
```

Comment:
maybe leave a comment that we're just reusing the built in `window.fetch` types for the module? I definitely completely missed what this was doing at first :)